django/db/models/sql/compiler.py

```diff
@@ -57,21 +57,23 @@ def __repr__(self):
             f"connection={self.connection!r} using={self.using!r}>"
         )
 
-    def setup_query(self):
```

Comment:
The `with_col_aliases=False` change that now propagate all the way to `get_select` could arguably be moved in a first commit. It basically allows for the `masked_aliases` call to properly take place.